team-builder/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import ReactDom from 'react-dom';
+import './App.css';
+import TeamMembersList from './components/TeamMembersList'
+import Form from './components/Form';
+
+function App() {
+
+  const [teamMembers, setTeamMembers] = useState([{id: 1, name:'Harry Potter', email:'hedwig@hogwarts.edu', role:'Auror'}]);
```

Comment:
You could have also created a component storing your "Data", imported it, and set it to your state.